Gemfile

```diff
@@ -31,7 +35,7 @@ gem "decidim", DECIDIM_VERSION
 
 # Remove this nokogiri forces version at any time but make sure that no __truncato_root__ text appears in the cards in general.
 # More exactly in comments in the homepage and in processes cards in the processes listing
-gem "nokogiri", "= 1.13.3"
+gem "nokogiri"
```

Comment:
Be careful!! this is an know error, can you check that not appear anymore? if error disappear, we can remove version in this gem and the comment.